# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1302181768 -7200
#      Thu Apr 07 15:09:28 2011 +0200
# Node ID baf7020fc0ea8bc405f426ca56e329ceba5c5501
# Parent  0861c8648be248cec968f3dde55326907fe0ddfd
Added folders and removed Makefile.

diff --git a/gui//Makefile b/gui//Makefile
deleted file mode 100644
--- a/gui//Makefile
+++ /dev/null
@@ -1,491 +0,0 @@
-#############################################################################
-# Makefile for building: bin/Quint
-# Generated by qmake (2.01a) (Qt 4.7.2) on: Do. Apr 7 15:04:11 2011
-# Project:  Quint.pro
-# Template: app
-# Command: /usr/bin/qmake -o Makefile Quint.pro
-#############################################################################
-
-####### Compiler, tools and options
-
-CC            = gcc
-CXX           = g++
-DEFINES       = -DHAVE_POSIX_OPENPT -DQT_NO_DEBUG -DQT_WEBKIT_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED
-CFLAGS        = -pipe -O2 -Wall -W -D_REENTRANT $(DEFINES)
-CXXFLAGS      = -pipe -O2 -Wall -W -D_REENTRANT $(DEFINES)
-INCPATH       = -I/usr/share/qt4/mkspecs/linux-g++ -I. -I/usr/include/qt4/QtCore -I/usr/include/qt4/QtGui -I/usr/include/qt4/QtWebKit -I/usr/include/qt4 -Isrc -Imoc-files
-LINK          = g++
-LFLAGS        = -Wl,-O1
-LIBS          = $(SUBLIBS)  -L/usr/lib -lQtWebKit -lQtGui -lQtCore -lpthread 
-AR            = ar cqs
-RANLIB        = 
-QMAKE         = /usr/bin/qmake
-TAR           = tar -cf
-COMPRESS      = gzip -9f
-COPY          = cp -f
-SED           = sed
-COPY_FILE     = $(COPY)
-COPY_DIR      = $(COPY) -r
-STRIP         = strip
-INSTALL_FILE  = install -m 644 -p
-INSTALL_DIR   = $(COPY_DIR)
-INSTALL_PROGRAM = install -m 755 -p
-DEL_FILE      = rm -f
-SYMLINK       = ln -f -s
-DEL_DIR       = rmdir
-MOVE          = mv -f
-CHK_DIR_EXISTS= test -d
-MKDIR         = mkdir -p
-
-####### Output directory
-
-OBJECTS_DIR   = object-files/
-
-####### Files
-
-SOURCES       = src/TerminalCharacterDecoder.cpp \
-		src/KeyboardTranslator.cpp \
-		src/Screen.cpp \
-		src/History.cpp \
-		src/BlockArray.cpp \
-		src/konsole_wcwidth.cpp \
-		src/ScreenWindow.cpp \
-		src/Emulation.cpp \
-		src/Vt102Emulation.cpp \
-		src/TerminalDisplay.cpp \
-		src/Filter.cpp \
-		src/Pty.cpp \
-		src/kpty.cpp \
-		src/k3process.cpp \
-		src/k3processcontroller.cpp \
-		src/Session.cpp \
-		src/ShellCommand.cpp \
-		src/QTerminalWidget.cpp \
-		src/TerminalMdiSubWindow.cpp \
-		src/MainWindow.cpp \
-		src/Quint.cpp moc-files/moc_ScreenWindow.cpp \
-		moc-files/moc_Emulation.cpp \
-		moc-files/moc_Vt102Emulation.cpp \
-		moc-files/moc_TerminalDisplay.cpp \
-		moc-files/moc_Filter.cpp \
-		moc-files/moc_Pty.cpp \
-		moc-files/moc_k3process.cpp \
-		moc-files/moc_k3processcontroller.cpp \
-		moc-files/moc_Session.cpp \
-		moc-files/moc_QTerminalWidget.cpp \
-		moc-files/moc_TerminalMdiSubWindow.cpp \
-		moc-files/moc_MainWindow.cpp
-OBJECTS       = object-files/TerminalCharacterDecoder.o \
-		object-files/KeyboardTranslator.o \
-		object-files/Screen.o \
-		object-files/History.o \
-		object-files/BlockArray.o \
-		object-files/konsole_wcwidth.o \
-		object-files/ScreenWindow.o \
-		object-files/Emulation.o \
-		object-files/Vt102Emulation.o \
-		object-files/TerminalDisplay.o \
-		object-files/Filter.o \
-		object-files/Pty.o \
-		object-files/kpty.o \
-		object-files/k3process.o \
-		object-files/k3processcontroller.o \
-		object-files/Session.o \
-		object-files/ShellCommand.o \
-		object-files/QTerminalWidget.o \
-		object-files/TerminalMdiSubWindow.o \
-		object-files/MainWindow.o \
-		object-files/Quint.o \
-		object-files/moc_ScreenWindow.o \
-		object-files/moc_Emulation.o \
-		object-files/moc_Vt102Emulation.o \
-		object-files/moc_TerminalDisplay.o \
-		object-files/moc_Filter.o \
-		object-files/moc_Pty.o \
-		object-files/moc_k3process.o \
-		object-files/moc_k3processcontroller.o \
-		object-files/moc_Session.o \
-		object-files/moc_QTerminalWidget.o \
-		object-files/moc_TerminalMdiSubWindow.o \
-		object-files/moc_MainWindow.o
-DIST          = /usr/share/qt4/mkspecs/common/g++.conf \
-		/usr/share/qt4/mkspecs/common/unix.conf \
-		/usr/share/qt4/mkspecs/common/linux.conf \
-		/usr/share/qt4/mkspecs/qconfig.pri \
-		/usr/share/qt4/mkspecs/modules/qt_webkit_version.pri \
-		/usr/share/qt4/mkspecs/features/qt_functions.prf \
-		/usr/share/qt4/mkspecs/features/qt_config.prf \
-		/usr/share/qt4/mkspecs/features/exclusive_builds.prf \
-		/usr/share/qt4/mkspecs/features/default_pre.prf \
-		/usr/share/qt4/mkspecs/features/release.prf \
-		/usr/share/qt4/mkspecs/features/default_post.prf \
-		/usr/share/qt4/mkspecs/features/warn_on.prf \
-		/usr/share/qt4/mkspecs/features/qt.prf \
-		/usr/share/qt4/mkspecs/features/unix/thread.prf \
-		/usr/share/qt4/mkspecs/features/moc.prf \
-		/usr/share/qt4/mkspecs/features/resources.prf \
-		/usr/share/qt4/mkspecs/features/uic.prf \
-		/usr/share/qt4/mkspecs/features/yacc.prf \
-		/usr/share/qt4/mkspecs/features/lex.prf \
-		Quint.pro
-QMAKE_TARGET  = Quint
-DESTDIR       = bin/
-TARGET        = bin/Quint
-
-first: all
-####### Implicit rules
-
-.SUFFIXES: .o .c .cpp .cc .cxx .C
-
-.cpp.o:
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o "$@" "$<"
-
-.cc.o:
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o "$@" "$<"
-
-.cxx.o:
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o "$@" "$<"
-
-.C.o:
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o "$@" "$<"
-
-.c.o:
-	$(CC) -c $(CFLAGS) $(INCPATH) -o "$@" "$<"
-
-####### Build rules
-
-all: Makefile $(TARGET)
-
-$(TARGET):  $(OBJECTS)  
-	@$(CHK_DIR_EXISTS) bin/ || $(MKDIR) bin/ 
-	$(LINK) $(LFLAGS) -o $(TARGET) $(OBJECTS) $(OBJCOMP) $(LIBS)
-
-Makefile: Quint.pro  /usr/share/qt4/mkspecs/linux-g++/qmake.conf /usr/share/qt4/mkspecs/common/g++.conf \
-		/usr/share/qt4/mkspecs/common/unix.conf \
-		/usr/share/qt4/mkspecs/common/linux.conf \
-		/usr/share/qt4/mkspecs/qconfig.pri \
-		/usr/share/qt4/mkspecs/modules/qt_webkit_version.pri \
-		/usr/share/qt4/mkspecs/features/qt_functions.prf \
-		/usr/share/qt4/mkspecs/features/qt_config.prf \
-		/usr/share/qt4/mkspecs/features/exclusive_builds.prf \
-		/usr/share/qt4/mkspecs/features/default_pre.prf \
-		/usr/share/qt4/mkspecs/features/release.prf \
-		/usr/share/qt4/mkspecs/features/default_post.prf \
-		/usr/share/qt4/mkspecs/features/warn_on.prf \
-		/usr/share/qt4/mkspecs/features/qt.prf \
-		/usr/share/qt4/mkspecs/features/unix/thread.prf \
-		/usr/share/qt4/mkspecs/features/moc.prf \
-		/usr/share/qt4/mkspecs/features/resources.prf \
-		/usr/share/qt4/mkspecs/features/uic.prf \
-		/usr/share/qt4/mkspecs/features/yacc.prf \
-		/usr/share/qt4/mkspecs/features/lex.prf \
-		/usr/lib/libQtWebKit.prl \
-		/usr/lib/libQtGui.prl \
-		/usr/lib/libQtCore.prl
-	$(QMAKE) -o Makefile Quint.pro
-/usr/share/qt4/mkspecs/common/g++.conf:
-/usr/share/qt4/mkspecs/common/unix.conf:
-/usr/share/qt4/mkspecs/common/linux.conf:
-/usr/share/qt4/mkspecs/qconfig.pri:
-/usr/share/qt4/mkspecs/modules/qt_webkit_version.pri:
-/usr/share/qt4/mkspecs/features/qt_functions.prf:
-/usr/share/qt4/mkspecs/features/qt_config.prf:
-/usr/share/qt4/mkspecs/features/exclusive_builds.prf:
-/usr/share/qt4/mkspecs/features/default_pre.prf:
-/usr/share/qt4/mkspecs/features/release.prf:
-/usr/share/qt4/mkspecs/features/default_post.prf:
-/usr/share/qt4/mkspecs/features/warn_on.prf:
-/usr/share/qt4/mkspecs/features/qt.prf:
-/usr/share/qt4/mkspecs/features/unix/thread.prf:
-/usr/share/qt4/mkspecs/features/moc.prf:
-/usr/share/qt4/mkspecs/features/resources.prf:
-/usr/share/qt4/mkspecs/features/uic.prf:
-/usr/share/qt4/mkspecs/features/yacc.prf:
-/usr/share/qt4/mkspecs/features/lex.prf:
-/usr/lib/libQtWebKit.prl:
-/usr/lib/libQtGui.prl:
-/usr/lib/libQtCore.prl:
-qmake:  FORCE
-	@$(QMAKE) -o Makefile Quint.pro
-
-dist: 
-	@$(CHK_DIR_EXISTS) object-files/Quint1.0.0 || $(MKDIR) object-files/Quint1.0.0 
-	$(COPY_FILE) --parents $(SOURCES) $(DIST) object-files/Quint1.0.0/ && $(COPY_FILE) --parents src/TerminalCharacterDecoder.h src/Character.h src/CharacterColor.h src/KeyboardTranslator.h src/ExtendedDefaultTranslator.h src/Screen.h src/History.h src/BlockArray.h src/konsole_wcwidth.h src/ScreenWindow.h src/Emulation.h src/Vt102Emulation.h src/TerminalDisplay.h src/Filter.h src/LineFont.h src/Pty.h src/kpty.h src/kpty_p.h src/k3process.h src/k3processcontroller.h src/Session.h src/ShellCommand.h src/QTerminalWidget.h src/TerminalMdiSubWindow.h src/MainWindow.h object-files/Quint1.0.0/ && $(COPY_FILE) --parents src/TerminalCharacterDecoder.cpp src/KeyboardTranslator.cpp src/Screen.cpp src/History.cpp src/BlockArray.cpp src/konsole_wcwidth.cpp src/ScreenWindow.cpp src/Emulation.cpp src/Vt102Emulation.cpp src/TerminalDisplay.cpp src/Filter.cpp src/Pty.cpp src/kpty.cpp src/k3process.cpp src/k3processcontroller.cpp src/Session.cpp src/ShellCommand.cpp src/QTerminalWidget.cpp src/TerminalMdiSubWindow.cpp src/MainWindow.cpp src/Quint.cpp object-files/Quint1.0.0/ && (cd `dirname object-files/Quint1.0.0` && $(TAR) Quint1.0.0.tar Quint1.0.0 && $(COMPRESS) Quint1.0.0.tar) && $(MOVE) `dirname object-files/Quint1.0.0`/Quint1.0.0.tar.gz . && $(DEL_FILE) -r object-files/Quint1.0.0
-
-
-clean:compiler_clean 
-	-$(DEL_FILE) $(OBJECTS)
-	-$(DEL_FILE) *~ core *.core
-
-
-####### Sub-libraries
-
-distclean: clean
-	-$(DEL_FILE) $(TARGET) 
-	-$(DEL_FILE) Makefile
-
-
-check: first
-
-mocclean: compiler_moc_header_clean compiler_moc_source_clean
-
-mocables: compiler_moc_header_make_all compiler_moc_source_make_all
-
-compiler_moc_header_make_all: moc-files/moc_ScreenWindow.cpp moc-files/moc_Emulation.cpp moc-files/moc_Vt102Emulation.cpp moc-files/moc_TerminalDisplay.cpp moc-files/moc_Filter.cpp moc-files/moc_Pty.cpp moc-files/moc_k3process.cpp moc-files/moc_k3processcontroller.cpp moc-files/moc_Session.cpp moc-files/moc_QTerminalWidget.cpp moc-files/moc_TerminalMdiSubWindow.cpp moc-files/moc_MainWindow.cpp
-compiler_moc_header_clean:
-	-$(DEL_FILE) moc-files/moc_ScreenWindow.cpp moc-files/moc_Emulation.cpp moc-files/moc_Vt102Emulation.cpp moc-files/moc_TerminalDisplay.cpp moc-files/moc_Filter.cpp moc-files/moc_Pty.cpp moc-files/moc_k3process.cpp moc-files/moc_k3processcontroller.cpp moc-files/moc_Session.cpp moc-files/moc_QTerminalWidget.cpp moc-files/moc_TerminalMdiSubWindow.cpp moc-files/moc_MainWindow.cpp
-moc-files/moc_ScreenWindow.cpp: src/Character.h \
-		src/CharacterColor.h \
-		src/ScreenWindow.h
-	/usr/bin/moc-qt4 $(DEFINES) $(INCPATH) src/ScreenWindow.h -o moc-files/moc_ScreenWindow.cpp
-
-moc-files/moc_Emulation.cpp: src/Emulation.h
-	/usr/bin/moc-qt4 $(DEFINES) $(INCPATH) src/Emulation.h -o moc-files/moc_Emulation.cpp
-
-moc-files/moc_Vt102Emulation.cpp: src/Emulation.h \
-		src/Screen.h \
-		src/Character.h \
-		src/CharacterColor.h \
-		src/History.h \
-		src/BlockArray.h \
-		src/Vt102Emulation.h
-	/usr/bin/moc-qt4 $(DEFINES) $(INCPATH) src/Vt102Emulation.h -o moc-files/moc_Vt102Emulation.cpp
-
-moc-files/moc_TerminalDisplay.cpp: src/Filter.h \
-		src/Character.h \
-		src/CharacterColor.h \
-		src/ColorTables.h \
-		src/TerminalDisplay.h
-	/usr/bin/moc-qt4 $(DEFINES) $(INCPATH) src/TerminalDisplay.h -o moc-files/moc_TerminalDisplay.cpp
-
-moc-files/moc_Filter.cpp: src/Character.h \
-		src/CharacterColor.h \
-		src/Filter.h
-	/usr/bin/moc-qt4 $(DEFINES) $(INCPATH) src/Filter.h -o moc-files/moc_Filter.cpp
-
-moc-files/moc_Pty.cpp: src/k3process.h \
-		src/Pty.h
-	/usr/bin/moc-qt4 $(DEFINES) $(INCPATH) src/Pty.h -o moc-files/moc_Pty.cpp
-
-moc-files/moc_k3process.cpp: src/k3process.h
-	/usr/bin/moc-qt4 $(DEFINES) $(INCPATH) src/k3process.h -o moc-files/moc_k3process.cpp
-
-moc-files/moc_k3processcontroller.cpp: src/k3processcontroller.h
-	/usr/bin/moc-qt4 $(DEFINES) $(INCPATH) src/k3processcontroller.h -o moc-files/moc_k3processcontroller.cpp
-
-moc-files/moc_Session.cpp: src/History.h \
-		src/BlockArray.h \
-		src/Character.h \
-		src/CharacterColor.h \
-		src/Session.h
-	/usr/bin/moc-qt4 $(DEFINES) $(INCPATH) src/Session.h -o moc-files/moc_Session.cpp
-
-moc-files/moc_QTerminalWidget.cpp: src/QTerminalWidget.h
-	/usr/bin/moc-qt4 $(DEFINES) $(INCPATH) src/QTerminalWidget.h -o moc-files/moc_QTerminalWidget.cpp
-
-moc-files/moc_TerminalMdiSubWindow.cpp: src/QTerminalWidget.h \
-		src/TerminalMdiSubWindow.h
-	/usr/bin/moc-qt4 $(DEFINES) $(INCPATH) src/TerminalMdiSubWindow.h -o moc-files/moc_TerminalMdiSubWindow.cpp
-
-moc-files/moc_MainWindow.cpp: src/MainWindow.h
-	/usr/bin/moc-qt4 $(DEFINES) $(INCPATH) src/MainWindow.h -o moc-files/moc_MainWindow.cpp
-
-compiler_rcc_make_all:
-compiler_rcc_clean:
-compiler_image_collection_make_all: ui-files/qmake_image_collection.cpp
-compiler_image_collection_clean:
-	-$(DEL_FILE) ui-files/qmake_image_collection.cpp
-compiler_moc_source_make_all:
-compiler_moc_source_clean:
-compiler_uic_make_all:
-compiler_uic_clean:
-compiler_yacc_decl_make_all:
-compiler_yacc_decl_clean:
-compiler_yacc_impl_make_all:
-compiler_yacc_impl_clean:
-compiler_lex_make_all:
-compiler_lex_clean:
-compiler_clean: compiler_moc_header_clean 
-
-####### Compile
-
-object-files/TerminalCharacterDecoder.o: src/TerminalCharacterDecoder.cpp src/TerminalCharacterDecoder.h \
-		src/Character.h \
-		src/CharacterColor.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/TerminalCharacterDecoder.o src/TerminalCharacterDecoder.cpp
-
-object-files/KeyboardTranslator.o: src/KeyboardTranslator.cpp src/KeyboardTranslator.h \
-		src/ExtendedDefaultTranslator.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/KeyboardTranslator.o src/KeyboardTranslator.cpp
-
-object-files/Screen.o: src/Screen.cpp src/Screen.h \
-		src/Character.h \
-		src/CharacterColor.h \
-		src/History.h \
-		src/BlockArray.h \
-		src/konsole_wcwidth.h \
-		src/TerminalCharacterDecoder.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/Screen.o src/Screen.cpp
-
-object-files/History.o: src/History.cpp src/History.h \
-		src/BlockArray.h \
-		src/Character.h \
-		src/CharacterColor.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/History.o src/History.cpp
-
-object-files/BlockArray.o: src/BlockArray.cpp src/BlockArray.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/BlockArray.o src/BlockArray.cpp
-
-object-files/konsole_wcwidth.o: src/konsole_wcwidth.cpp src/konsole_wcwidth.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/konsole_wcwidth.o src/konsole_wcwidth.cpp
-
-object-files/ScreenWindow.o: src/ScreenWindow.cpp src/ScreenWindow.h \
-		src/Character.h \
-		src/CharacterColor.h \
-		src/Screen.h \
-		src/History.h \
-		src/BlockArray.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/ScreenWindow.o src/ScreenWindow.cpp
-
-object-files/Emulation.o: src/Emulation.cpp src/Emulation.h \
-		src/KeyboardTranslator.h \
-		src/Screen.h \
-		src/Character.h \
-		src/CharacterColor.h \
-		src/History.h \
-		src/BlockArray.h \
-		src/TerminalCharacterDecoder.h \
-		src/ScreenWindow.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/Emulation.o src/Emulation.cpp
-
-object-files/Vt102Emulation.o: src/Vt102Emulation.cpp src/Vt102Emulation.h \
-		src/Emulation.h \
-		src/Screen.h \
-		src/Character.h \
-		src/CharacterColor.h \
-		src/History.h \
-		src/BlockArray.h \
-		src/KeyboardTranslator.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/Vt102Emulation.o src/Vt102Emulation.cpp
-
-object-files/TerminalDisplay.o: src/TerminalDisplay.cpp src/TerminalDisplay.h \
-		src/Filter.h \
-		src/Character.h \
-		src/CharacterColor.h \
-		src/ColorTables.h \
-		src/konsole_wcwidth.h \
-		src/ScreenWindow.h \
-		src/TerminalCharacterDecoder.h \
-		src/LineFont.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/TerminalDisplay.o src/TerminalDisplay.cpp
-
-object-files/Filter.o: src/Filter.cpp src/Filter.h \
-		src/Character.h \
-		src/CharacterColor.h \
-		src/TerminalCharacterDecoder.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/Filter.o src/Filter.cpp
-
-object-files/Pty.o: src/Pty.cpp src/Pty.h \
-		src/k3process.h \
-		src/kpty.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/Pty.o src/Pty.cpp
-
-object-files/kpty.o: src/kpty.cpp src/kpty_p.h \
-		src/kpty.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/kpty.o src/kpty.cpp
-
-object-files/k3process.o: src/k3process.cpp src/k3process.h \
-		src/k3processcontroller.h \
-		src/kpty.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/k3process.o src/k3process.cpp
-
-object-files/k3processcontroller.o: src/k3processcontroller.cpp src/k3processcontroller.h \
-		src/k3process.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/k3processcontroller.o src/k3processcontroller.cpp
-
-object-files/Session.o: src/Session.cpp src/Session.h \
-		src/History.h \
-		src/BlockArray.h \
-		src/Character.h \
-		src/CharacterColor.h \
-		src/Pty.h \
-		src/k3process.h \
-		src/TerminalDisplay.h \
-		src/Filter.h \
-		src/ColorTables.h \
-		src/ShellCommand.h \
-		src/Vt102Emulation.h \
-		src/Emulation.h \
-		src/Screen.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/Session.o src/Session.cpp
-
-object-files/ShellCommand.o: src/ShellCommand.cpp src/ShellCommand.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/ShellCommand.o src/ShellCommand.cpp
-
-object-files/QTerminalWidget.o: src/QTerminalWidget.cpp src/QTerminalWidget.h \
-		src/Session.h \
-		src/History.h \
-		src/BlockArray.h \
-		src/Character.h \
-		src/CharacterColor.h \
-		src/TerminalDisplay.h \
-		src/Filter.h \
-		src/ColorTables.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/QTerminalWidget.o src/QTerminalWidget.cpp
-
-object-files/TerminalMdiSubWindow.o: src/TerminalMdiSubWindow.cpp src/TerminalMdiSubWindow.h \
-		src/QTerminalWidget.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/TerminalMdiSubWindow.o src/TerminalMdiSubWindow.cpp
-
-object-files/MainWindow.o: src/MainWindow.cpp src/MainWindow.h \
-		src/TerminalMdiSubWindow.h \
-		src/QTerminalWidget.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/MainWindow.o src/MainWindow.cpp
-
-object-files/Quint.o: src/Quint.cpp src/MainWindow.h
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/Quint.o src/Quint.cpp
-
-object-files/moc_ScreenWindow.o: moc-files/moc_ScreenWindow.cpp 
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/moc_ScreenWindow.o moc-files/moc_ScreenWindow.cpp
-
-object-files/moc_Emulation.o: moc-files/moc_Emulation.cpp 
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/moc_Emulation.o moc-files/moc_Emulation.cpp
-
-object-files/moc_Vt102Emulation.o: moc-files/moc_Vt102Emulation.cpp 
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/moc_Vt102Emulation.o moc-files/moc_Vt102Emulation.cpp
-
-object-files/moc_TerminalDisplay.o: moc-files/moc_TerminalDisplay.cpp 
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/moc_TerminalDisplay.o moc-files/moc_TerminalDisplay.cpp
-
-object-files/moc_Filter.o: moc-files/moc_Filter.cpp 
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/moc_Filter.o moc-files/moc_Filter.cpp
-
-object-files/moc_Pty.o: moc-files/moc_Pty.cpp 
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/moc_Pty.o moc-files/moc_Pty.cpp
-
-object-files/moc_k3process.o: moc-files/moc_k3process.cpp 
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/moc_k3process.o moc-files/moc_k3process.cpp
-
-object-files/moc_k3processcontroller.o: moc-files/moc_k3processcontroller.cpp 
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/moc_k3processcontroller.o moc-files/moc_k3processcontroller.cpp
-
-object-files/moc_Session.o: moc-files/moc_Session.cpp 
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/moc_Session.o moc-files/moc_Session.cpp
-
-object-files/moc_QTerminalWidget.o: moc-files/moc_QTerminalWidget.cpp 
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/moc_QTerminalWidget.o moc-files/moc_QTerminalWidget.cpp
-
-object-files/moc_TerminalMdiSubWindow.o: moc-files/moc_TerminalMdiSubWindow.cpp 
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/moc_TerminalMdiSubWindow.o moc-files/moc_TerminalMdiSubWindow.cpp
-
-object-files/moc_MainWindow.o: moc-files/moc_MainWindow.cpp 
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o object-files/moc_MainWindow.o moc-files/moc_MainWindow.cpp
-
-####### Install
-
-install:   FORCE
-
-uninstall:   FORCE
-
-FORCE:
-
